crates/proc-macro-srv-cli/src/main.rs

```diff
@@ -31,7 +31,7 @@ fn main() -> std::io::Result<()> {
             clap::Arg::new("format")
                 .long("format")
                 .action(clap::ArgAction::Set)
-                .default_value("json")
```

Comment:
The default should stay as json or this will break older clients that don't support postcard